.drone.yml

```diff
@@ -9,7 +9,7 @@ platform:
 steps:
 - name: build
   pull: default
-  image: rancher/dapper:1.10.3
```

Comment:
Why is this being downgraded?

Reply:
I think the versioning changed overtime, the latest is `v0.6.0`:

![image](https://github.com/rancher/cli/assets/5452977/a786f737-5180-4153-a620-2714db6a1fcf)
![image](https://github.com/rancher/cli/assets/5452977/b05da5ea-aebc-4fb2-a30b-149c0949bddb)

This aligns with that repository Git tags.

Reply:
This ^ is right. [v0.6.0](https://github.com/rancher/dapper/releases/tag/v0.6.0) is from Jul 7, 2022 and 1.10.3 is not even in current [Dapper's repo](https://github.com/rancher/dapper/releases?page=2).